lang/java/reef-examples/src/main/java/org/apache/reef/examples/hello/HelloTaskDelayed.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.reef.examples.hello;
+
+import org.apache.reef.task.Task;
+
+import javax.inject.Inject;
+
+/**
+ * A 'hello REEF' Task.
+ */
+public final class HelloTaskDelayed implements Task {
```

Comment:
It looks like there is a bug in local runtime. Looks like a race condition between evaluator communication creation and task completion, so without the delay the task actually exits before communication channel is established and we cannot send notification back. This results in unclear shutdown, just wanted to see clear logs for this runtime. I agree that we need to fix this but don't want to wait for this. Will create JIRA for this issue.